README.md

```diff
@@ -184,9 +184,9 @@ Patches are encouraged, and may be submitted by forking this project and submitt
 
 Copyright 2015 Google, Inc.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+Licensed under the [Apache License, Version 2.0[(https://github.com/GoogleChrome
+/sw-toolbox/blob/master/README.md)] (the "License"); you may not use this file
```

Comment:
I think this should link to https://github.com/GoogleChrome/sw-toolbox/blob/master/LICENSE, which can be simplified in GitHub Markdown to `[Apache License, Version 2.0](LICENSE)`.